api/filter/http_fault.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter;
+
+import "api/rds.proto";
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/wrappers.proto";
+
+// Delay specification is used to inject latency into the rpc operations.
+message Delay {
+  // Delay type to use (fixed|exponential|..). Currently, only fixed delay (step function) is supported.
+  string type = 1;
+
+  // An integer between 0-100 indicating the percentage of operations
+  // on which the delay will be injected.
+  google.protobuf.UInt32Value percent = 2;
+
+  oneof http_delay_type {
+    // Add a fixed delay before forwarding the operation upstream. Format: 1h/1m/1s/1ms. MUST be >=1ms.
```

Comment:
There is no format for `Duration`s..

Reply:
Yup. But this was intentional. For the doc-gen.. Coz it does not tell people that Duration is just a string and that people can specify 1h/1m/1s. When we generated docs for Istio, this was  one of the issues. So, I went back to all protos, and manually added this, so that people reading docs knew what values to put.

Reply:
`Duration` is only a string in JSON/YAML form. We will eventually be generating docs for {proto3, JSON, YAML}, so this interpretation only applies to some readings. I'm surprised that docgen can't inline the documentations from `Duration`, which is where you would expect to find this kind of explanation.

I think a better way to put it might be "See https://developers.google.com/protocol-buffers/docs/proto3#json for the JSON/YAML Duration mapping.".